logic/src/commonMain/kotlin/com/wire/kalium/logic/data/wireuser/model/WireUser.kt

```diff
@@ -1,12 +1,12 @@
-package com.wire.kalium.logic.data.publicuser.model
+package com.wire.kalium.logic.data.wireuser.model
 
 import com.wire.kalium.logic.data.user.User
 import com.wire.kalium.logic.data.user.UserAssetId
 import com.wire.kalium.logic.data.user.UserId
 
-data class PublicUser(
+data class WireUser(
```

Comment:
I'd either keep the name `PublicUser` for now or if want to have generic user model let's just call it `User`

Reply:
we already have User as abstract :S I will change it to PublicUser as for now